docs/apis/plugintypes/mod_workshop/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Workshop
+tags: []
+---
+:::info
+Work in progress
+:::
+
+import { Since } from '@site/src/components';
+
+<Since versions={["2.0"]} />
```

Comment:
```suggestion
```

There's no point really including any really old since tags. No-one should be using 1.9- and _no-one_ should be developing for it.

We're trying to update our dev docs to be version specific to avoid having to spend a lot of time understanding which items only relate to an ancient version of Moodle.